### ISSUE TITLE
Screen becomes strange when using native resolution 1366x768 60Hz

### ISSUE THREAD

**Issue by rafik24** [mrworf/photoframe#125]
Hi MrWorf,

One again thanks for this great project !

I have an LCD driver board and I cannot setup the max resolution 1366x60 as the picture becomes weird please see === wrong link ===.

Setting the resolution lower is ok but the picture quality is not great.

I have tried installing a desktop on the RPI and it works just well on 1366x768 60Hz plays videos and all, then stopping the window manager and starting the frame service still has the same weird picture issue.

I have played around with most setting in the /boot/config.txt file in vain.

tried also to look at fbset and tvservice which show the same exact mode and informationusing the frame or X as per below:

Running frame 1368x768 60Hz

mode "1366x768"
    geometry 1366 768 1366 768 32
    timings 0 0 0 0 0 0 0
    rgba 8/16,8/8,8/0,8/24
endmode

Frame buffer device information:
    Name        : BCM2708 FB
    Address     : 0x1e7f2000
    Size        : 4227072
    Type        : PACKED PIXELS
    Visual      : TRUECOLOR
    XPanStep    : 1
    YPanStep    : 1
    YWrapStep   : 0
    LineLength  : 5504
    Accelerator : No

# tvservice -s 

state 0xa [HDMI DMT (81) RGB full 16:9], 1366x768 @ 60.00Hz, progressive

Can you please help ?

Thanks,
Rafik

**Comment by mrworf** [comment]
Can you provide more details? Maybe a photo of the screen when it's OK and when it's weird so I know the difference (current link is no bueno, pointing to the wrong item, so I removed it). Also what screen you're using.

Does the boot console look ok? If it does, can you dump out the fbset for that too?

**Comment by rafik24** [comment]
Hi mrworf,

Thank you for looking at this.

I come with good news, the issue is not with photoframe but with my LCD controller interface NT68676.2 for reference.

After a lot of digging I discovered the EDID from the controller does not reference the correct mode for my screen.

While my screen native resolution is 1366x768 the EDID does show this mode but references 1200x768 when this mode is selected.

In order to avoid this and add a custom mode I had to add the following line to /boot/config.txt

hdmi_cvt=1366 768 60 3 0 0 1

after rebbot this creates a new DMT mode that can be seen in tvservice -s

this mode can be selected as default as per below:

hdmi_group=2
hdmi_mode=87 

Where 87 was the number assigned to the custom mode.

Adding the picture URL for reference hopping it would help someone one day :-)
https://ibb.co/98JzHh1

Many Thanks,
Rafik
